userspace/libscap/scap.c

```diff
@@ -1091,6 +1091,11 @@ uint64_t scap_max_buf_used(scap_t* handle)
 int32_t scap_next(scap_t* handle, OUT scap_evt** pevent, OUT uint16_t* pcpuid)
 {
 	int32_t res = SCAP_FAILURE;
+	if(handle == NULL)
+	{
+		return res;
+	}
+
```

Comment:
This is the hot path so I suppose we don't want to check the handle here. If a consumer comes far enough to call `scap_next` (obviously ignoring all previous failures), I'd say let them crash :D